Automattic/wp-calypso
# Jetpack Cloud: add "threats" badge to the sidebar

#### Changes proposed in this Pull Request

- The goal of this PR is to add a badge to the Scanner sidebar menu item. It should display the number of threats found in the last scan process.

#### Important note

Threats are fetched from a fake selector since, at this point, we haven't integrated real endpoints to Scan.

#### Testing instructions

- Navigate to http://jetpack.cloud.localhost:3000/scan/<yourSiteSlug>
- Open the Scan menu item
- See the badge that displays the number of threats

See 1151678672052943-as-1165876793353297.

#### Expected result
<img width="267" alt="image" src="https://user-images.githubusercontent.com/3418513/77002886-0f5d9680-693b-11ea-9b59-2db78cf281c8.png">

#### Current result
One threat
<img width="268" alt="image" src="https://user-images.githubusercontent.com/3418513/77003537-5009df80-693c-11ea-817d-167beec7a1ac.png">

Two threats
<img width="272" alt="image" src="https://user-images.githubusercontent.com/3418513/77002871-04a30180-693b-11ea-9926-4bf2feeb5312.png">
